CONTRIBUTING.md

```
@@ -157,9 +156,9 @@ and set the values of the variables according to your environment.
 >     browser testing, but remote development and production deployments need
 >     HTTPS for browser Push API subscriptions.
 >
->  -  `KV_URL` must point to Redis when running the standalone GraphQL API and
->     worker.  A file-backed store is only safe for a single process, such as
->     the legacy web stack or one-off command-line utilities.
+>  -  `KV_URL` may point to a file when running only `mise run dev:graphql`.
```

markdownlint-cli2 (0.23.0)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)